homeassistant/components/device_tracker/fritz.py

```diff
@@ -0,0 +1,94 @@
+""" Supports scanning a FritzBox router"""
+import logging
+from datetime import timedelta
+from homeassistant.const import CONF_HOST, CONF_USERNAME, CONF_PASSWORD
+from homeassistant.helpers import validate_config
+from homeassistant.util import Throttle
+from homeassistant.components.device_tracker import DOMAIN
+
+
+# Return cached results if last scan was less then this time ago
+MIN_TIME_BETWEEN_SCANS = timedelta(seconds=5)
+
+_LOGGER = logging.getLogger(__name__)
+
+REQUIREMENTS = ['fritzconnection==0.4.6']
+
+def get_scanner(hass, config):
+    """ Validates config and returns FritzBoxScanner obj"""
+    if not validate_config(config,
+                           {DOMAIN: [CONF_HOST, CONF_USERNAME, CONF_PASSWORD]},
+                           _LOGGER):
+        return None
+    scanner = FritzBoxScanner(config[DOMAIN])
+    return scanner if scanner.success_init else None
+
+
+# pylint: disable=too-many-instance-attributes
+class FritzBoxScanner():
+    """ This class queries a FritzBox router. Adapted from luci.py and
+    using fritzconnection as comunication backend with the router
+
+    The API describtion can be found under:
+    https://pypi.python.org/pypi/fritzconnection/0.4.6
+
+    This scanner asks the fritzbox for its known hosts and returns
+    theis state (on, or off)
+
+    Due to a bug in the fritzbox api (router side), not more than
+    16 hosts are returned."""
+    def __init__(self, config):
+        self.last_results = []
+
+        try:
+            import fritzconnection as fc
+        except ImportError:
+            # pylint: disable=line-too-long
+            _LOGGER.exception('Failed to import fritzconnection. Did you install fritzconnection with the bugfix from "https://bitbucket.org/Fettlaus/fritzconnection"') # flake8: noqa
```

Comment:
Don't disable line too long. This line is actually too long. Please break it up. You can use [implicit string concatenation](http://stackoverflow.com/questions/2058925/how-can-i-break-up-this-long-line-in-python).